app/config/config.yml

```diff
@@ -86,10 +85,12 @@ assetic:
             output: css/min.css
         common_css:
             inputs:
-                - "bundles/website/css/common.css"
+                - "@WebsiteBundle/Resources/public/css/common.css"
```

Comment:
@chilic may I combine `min.css` and `common.css` to `common.css` just to decrease number of used css files ?